### ISSUE TITLE
Feedback - [Accessibility]: Missing alt content on the images

### ISSUE THREAD

**Issue by rollybueno** [WordPress/Learn#3181]
## Type of feedback

//content

## Description

On this learn page, https://learn.wordpress.org/lesson/developing-with-hooks/, it has 2 images content that both have empty alt attribute to the <img> tag which causing accessibility issue. 

## Step-by-step reproduction instructions (optional)

1. Open https://learn.wordpress.org/lesson/developing-with-hooks/
2. Navigate to the 2 images on the dev console.
3. You should see there's alt attribute for these 2, but empty.

## Screenshots or screen recording (optional)

![Image](https://github.com/user-attachments/assets/a5bad685-cb1f-4f1f-9848-892a2360f6eb)

![Image](https://github.com/user-attachments/assets/389460a6-bf95-4edc-8f65-ac3d21b6835e)

**Comment by rjekic** [comment]
The alt attribute isn't always required. In this case, the images are already descriptive, and providing alternative text might actually create confusion for users with visual impairments. Both images visually represent content that's already explained in the surrounding text. However, to err on the side of caution, we can include alt text for both images. We'll just need to agree on what that text should be. I attempted to come up with something appropriate, but haven’t had much success so far.

**Comment by rollybueno** [comment]
My suggestions:

- For the assert callback: _Flowchart showing data flow from an external plugin's filter through user-defined callbacks and your code, illustrating interactions between External plugin code, Userland code, and Your code._

- For the handling filter: _Diagram showing data flow from a filter through a series of user-defined callbacks, illustrating how 'Your code' interacts with 'Userland code' via Callback 1 to Callback 4._

Let me know your thoughts

**Comment by rjekic** [comment]
The alt text aims to explain the image content to people with visual impairment. We should be careful and try not to confuse them. In this case, I think that we can use the provided solutions, maybe to tweak them a bit, if it seems confusing to you all. It is readable and understandable to me, but I would like to make sure we have the best version.

Thank you @rollybueno 

cc @devmuhib009

**Comment by rjekic** [comment]
@rollybueno 

I've tried to update the alt with the proposed solutions, but the Wave evaluation tool gives 2 errors for the long alternative text on both images.

![Image](https://github.com/user-attachments/assets/fc0bd9e3-60e3-46a7-b230-25409568238a)

![Image](https://github.com/user-attachments/assets/a519a8e9-0517-4f5b-9c46-4abb7403ade1)

This is the shortest version I could get for both images. I've added it to the lesson.

_For the handling filter:_  Flowchart showing data passed through filter and user callbacks.
_For the assert callback:_ Flowchart showing filter applying multiple callbacks, including user and plugin code.

If anyone finds a better solution, I will update the alt text on both images.

@devmuhib009

**Comment by rollybueno** [comment]
Hey @rjekic - thanks! I think your alt text version is okay. I checked https://learn.wordpress.org/lesson/developing-with-hooks/ and both images now have alt text. We can close this.

Thanks!
